assembl/static2/js/app/components/debate/thread/topPostForm.jsx

```diff
@@ -1,148 +1,193 @@
-                maxLength={TEXT_INPUT_MAX_LENGTH}
-                handleTxtChange={this.handleSubjectChange}
-                handleInputFocus={this.handleInputFocus}
-              />
-              <div className={this.state.isActive ? 'margin-m' : 'hidden'}>
-                <RichTextEditor
-                  rawContentState={this.state.body}
-                  handleInputFocus={this.handleInputFocus}
-                  maxLength={TEXT_AREA_MAX_LENGTH}
-                  placeholder={I18n.t('debate.insert')}
-                  updateContentState={this.updateBody}
-                />
-                <Button className="button-cancel button-dark btn btn-default left margin-l" onClick={this.resetForm}>
-                  <Translate value="cancel" />
-                </Button>
-                <Button
-                  className="button-submit button-dark btn btn-default right margin-l"
-                  onClick={this.createTopPost}
-                  style={{ marginBottom: '30px' }}
-                  disabled={this.state.submitting}
-                >
-                  <Translate value="debate.post" />
-                </Button>
-              </div>
-            </FormGroup>
+          <div className="start-discussion">
+            <h3 className="dark-title-3 no-margin">
+              <Translate value="debate.thread.startDiscussion" />
+            </h3>
           </div>
-        </Col>
-        <Col xs={0} sm={1} md={2} />
-      </Row>
-    );
-  }
-}
+        </div>
+      </Col>
+      <Col xs={12} sm={7} md={6} className="no-padding">
+        <div className="form-container">
+          <FormGroup>
+            <TextInputWithRemainingChars
+              value={subject}
+              label={I18n.t('debate.subject')}
+              maxLength={TEXT_INPUT_MAX_LENGTH}
+              handleTxtChange={handleSubjectChange}
+              handleInputFocus={handleInputFocus}
+              remainingChars={subjectTopPostRemainingChars}
+            />
+            <div className={isFormActive ? 'margin-m' : 'hidden'}>
+              <TxtAreaWithRemainingChars
+                value={body}
+                label={I18n.t('debate.insert')}
+                maxLength={TEXT_AREA_MAX_LENGTH}
+                rows={TEXT_AREA_ROWS}
+                handleTxtChange={handleBodyChange}
+                remainingChars={bodyTopPostRemainingChars}
+              />
+              <Button className="button-cancel button-dark btn btn-default left margin-l" onClick={resetForm}>
+                <Translate value="cancel" />
+              </Button>
+              <Button
+                className="button-submit button-dark btn btn-default right margin-l"
+                onClick={createTopPost}
+                style={{ marginBottom: '30px' }}
+              >
+                <Translate value="debate.post" />
+              </Button>
+            </div>
+          </FormGroup>
+        </div>
+      </Col>
+      <Col xs={0} sm={1} md={2} />
```

Comment:
You can use mdOffset and smOffset. I changed it in https://github.com/assembl/assembl/commit/c8f4ab9f3703f6b2de764041a8e4d1070d9eff6a